tests/test_now_playing.py

```
@@ -0,0 +1,449 @@
+"""Tests for the now-playing subsystem: models, manager, events, dedupe."""
+
+from __future__ import annotations
+
+import asyncio
+import io
+import time
+from collections import OrderedDict
```

Flake8 (7.3.0)

[error] 'collections.OrderedDict' imported but unused

(F401)